script/run-unit-test.js

```diff
@@ -44,12 +49,83 @@ const testRunner = options.coverage ? coveragePath : mochaPath;
 const mochaOpts =
   'src/platform/testing/unit/mocha.opts src/platform/testing/unit/helper.js';
 
-// Otherwise, run the command
-runCommand(
-  `LOG_LEVEL=${options[
-    'log-level'
-  ].toLowerCase()} ${testRunner} --max-old-space-size=4096 --opts ${mochaOpts} --recursive ${options.path
-    .map(p => `'${p}'`)
-    .join(' ')}`,
-  options.coverage ? null : 0,
-);
+const logLevel = options['log-level'].toLowerCase();
+const forcedExitCode = options.coverage ? null : 0;
+const testDirectories = options.path;
+
+if (testDirectories[0] === defaultPath) {
+  const allUnitTests = glob.sync(defaultPath);
```

Comment:
```suggestion
  // array of unit test paths 
  const allUnitTestPaths = glob.sync(defaultPath);
```
just to save someone having to look up the glob docs